crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -372,7 +372,7 @@ public static bool RunningWithoutGPU
 
         public static int sp_crestTime = Shader.PropertyToID("_CrestTime");
         readonly int sp_texelsPerWave = Shader.PropertyToID("_TexelsPerWave");
-        readonly int sp_oceanCenterPosWorld = Shader.PropertyToID("_OceanCenterPosWorld");
+        public static int sp_oceanCenterPosWorld = Shader.PropertyToID("_OceanCenterPosWorld");
```

Comment:
Are we able to keep readonly?

Reply:
it can. i tried to make others readonly but some of them are written in InitStatics and some not. i dont understand/remember why its like this. might be worth a comment in InitStatics if we can recall why some need to be written and some not.